components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -471,6 +471,9 @@ void AdsServiceImpl::RemoveFirstLaunchNotification() {
       rewards_service->GetNotificationService();
   rewards_notification_service->DeleteNotification(
       kRewardsNotificationAdsLaunch);
+
+  profile_->GetPrefs()->SetBoolean(
```

Comment:
Function name uses Remove, so can we change Delete/Deleted to Remove/Removed for consistency throughout change. Other than this looks great to me 👍

Reply:
@tmancey agreed, updated to use `removed`